jarbas/core/management/commands/companies.py

```diff
@@ -23,6 +24,8 @@ def handle(self, *args, **options):
 
         self.save_companies()
 
+
+
```

Comment:
We don't need these extra spaces: according to PEP8 1 line is fine to separate class methods.